mesonbuild/compilers/c_function_attributes.py

```diff
@@ -93,8 +93,15 @@
     'visibility': '''
         int foo_def(void) __attribute__((visibility("default")));
         int foo_hid(void) __attribute__((visibility("hidden")));
-        int foo_int(void) __attribute__((visibility("internal")));
-        int foo_pro(void) __attribute__((visibility("protected")));''',
+        int foo_int(void) __attribute__((visibility("internal")));''',
+    'visibility:default':
+        'int foo(void) __attribute__((visibility("default")));',
+    'visibility:hidden':
+        'int foo(void) __attribute__((visibility("hidden")));',
+    'visibility:internal':
+       'int foo(void) __attribute__((visibility("internal")));',
```

Comment:
__[Flake8]__ [E131] continuation line unaligned for hanging indent ([link](https://sider.review/gh/repos/19784232/pulls/5531#issue-4274016))

<sub>posted by Sider</sub>